### PR TITLE
MC-17224: Bucket content docs

### PR DESCRIPTION
**Pull request by nicholasturk**
#### Changes made
Added doc section for getting objects.

#### Related PRs
- [Bucket content s3 sdk](https://github.com/cloudops/cloudmc-s3-sdk/pull/12)

### CODE REVIEW

File: source/includes/aws/_objects.md [hunk 1]
```diff
@@ -2,7 +2,100 @@
...
+| `name`<br/>*string*               | The name of the object.                                                                                                                                                     
+| `fullPath`<br/>*string*               | The actual key of the object inside the bucket, which is represented as a full path.                                                                                                                                                    |
+| `size`<br/>*integer*     | The size of the object in bytes.                                                                                                                                                     |
+| `storageClass`<br/>*string*            | The Type of storage class on the object.                                                                                                                                                      |
+| `lastModified`<br/>*string*       | The date the last time the object was updated.
```
Comment (lamminade) [inline source/includes/aws/_objects.md]:
```suggestion
| `lastModified`<br/>*string*       | The date the object was last updated.
```
maybe? just a lil awkward

File: source/includes/aws/_objects.md [hunk 2]
```diff
@@ -2,7 +2,100 @@
...
+| `displaySize`<br/>*string*       | The size of the object represented as an object in either KB/MB/GB.
+| `contentType`<br/>*string*       | The type of file.
+| `eTag`<br/>*string*       | The eTag of the object generated by S3 during initial upload.
+| `isFolder`<br/>*boolean*       | Boolean denoting whether the object actr as a folder. This is always true when the name ends with a '/' character.
```
Comment (lamminade) [inline source/includes/aws/_objects.md]:
```suggestion
| `isFolder`<br/>*boolean*       | Boolean denoting whether the object acts as a folder. This is always true when the name ends with a '/' character.
```

File: source/includes/aws/_objects.md [hunk 3]
```diff
@@ -2,7 +2,100 @@
...
+| `bucketName`<br/>*string*       | The name of the bucket the object exists within.
+| `region`<br/>*string*       | The name of the region the object exists within.
+| `isVirtual`<br/>*string*       | Boolean denoting whether the folder actually exists (there exists a key in the bucket named 'folder/'), or just because it's part of the path of a complete object path ('folder/object'). In this request, if nested=true, it will not return any virtual folders due to the nature of obtaining common prefixes and nested objects.
```
Comment (lamminade) [inline source/includes/aws/_objects.md]: this is a little confusing to me

File: source/includes/aws/_objects.md [hunk 4]
```diff
@@ -2,7 +2,100 @@
...
+
+| Mandatory Query Paramaters                        | &nbsp;                                                                                                                                                                                                                   |
```
Comment (lamminade) [inline source/includes/aws/_objects.md]:
```suggestion
| Required Query Paramaters                        | &nbsp;                                                                                                                                                                                                                   |
```
to keep it consistent within the docs

File: source/includes/aws/_objects.md (hunk 3 above)
Comment (nicholasturk) [reply source/includes/aws/_objects.md]: yeah, i think im just gonna omit that.

Comment (nicholasturk) [reply source/includes/aws/_objects.md]: actually im not sure its kind of import but i don't know how to explain it any better

File: source/includes/aws/_objects.md (hunk 2 above)
Comment (nicholasturk) [reply source/includes/aws/_objects.md]: https://github.com/cloudops/cloudmc-api-docs/commit/be42623c5dd36eda0a92e0993af4077e0ada4b37

File: source/includes/aws/_objects.md (hunk 1 above)
Comment (nicholasturk) [reply source/includes/aws/_objects.md]: https://github.com/cloudops/cloudmc-api-docs/commit/be42623c5dd36eda0a92e0993af4077e0ada4b37

File: source/includes/aws/_objects.md (hunk 4 above)
Comment (nicholasturk) [reply source/includes/aws/_objects.md]: https://github.com/cloudops/cloudmc-api-docs/commit/be42623c5dd36eda0a92e0993af4077e0ada4b37

File: source/includes/aws/_objects.md [hunk 5]
```diff
@@ -2,7 +2,100 @@
...
+| `name`<br/>*string*               | The name of the object.                                                                                                                                                     
+| `fullPath`<br/>*string*               | The actual key of the object inside the bucket, which is represented as a full path.                                                                                                                                                    |
+| `size`<br/>*integer*     | The size of the object in bytes.                                                                                                                                                     |
+| `storageClass`<br/>*string*            | The Type of storage class on the object.                                                                                                                                                      |
```
Comment (lamminade) [inline source/includes/aws/_objects.md]: does Type need to be capitalized? 
```suggestion
| `storageClass`<br/>*string*            | The type of storage class on the object.                                                                                                                                                      |
```

File: source/includes/aws/_objects.md (hunk 3 above)
Comment (nicholasturk) [reply source/includes/aws/_objects.md]: https://github.com/cloudops/cloudmc-api-docs/commit/69a44b2bf6b96a5926d64a9ce4ea39c163147e25

File: source/includes/aws/_objects.md (hunk 5 above)
Comment (nicholasturk) [reply source/includes/aws/_objects.md]: https://github.com/cloudops/cloudmc-api-docs/commit/69a44b2bf6b96a5926d64a9ce4ea39c163147e25

**Review by lamminade — APPROVED** [review]

**Pull request merged**